### PR TITLE
Style caption settings for better usability

### PR DESCRIPTION
**Pull request by aaron-collier**
Part of #1498 

This is a first step in applying the design for the caption settings dialog. This cleans up the existing dialog so that it is usable, a future PR will be necessary IF the design can be made applicable to the provided videojs html.

Screenshot 2023-11-16 at 3 42 39 PM

### CODE REVIEW

**Review by jcoyne — APPROVED** [review]

File: app/assets/stylesheets/media.scss [hunk 1]
```diff
@@ -421,3 +421,31 @@
...
+  border-radius: var(--modal-border-radius);
+  box-shadow: 5px 10px 5px gray;
+  font-size: 1em;
+
+}
+
+.vjs-track-settings-controls button {
+  border: 1px solid black;
+  border-radius: var(--modal-border-radius);
+  box-shadow: 2px 3px 3px gray;
```
Comment (jcoyne) [inline app/assets/stylesheets/media.scss:444]: I believe this is the only button in the site that has a shadow.  Do we want this to be so unique?

Comment (aaron-collier) [reply app/assets/stylesheets/media.scss:444]: I could totally remove that, I was just trying to make it a bit more obviously a button.

File: app/assets/stylesheets/media.scss [hunk 2]
```diff
@@ -421,3 +421,30 @@
     }
   }
 }
+
+.video-js .vjs-text-track-settings {
+  background: white !important;
+  /* overflow: auto; */
```
Comment (mjgiarlo) [inline app/assets/stylesheets/media.scss:427]: cruft?

Comment (aaron-collier) [reply app/assets/stylesheets/media.scss]: Yup - I thought I'd caught all my comments, thanks, removed.

**Review by mjgiarlo — APPROVED** [review]

**Pull request merged**